tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/ProbesTests.cs

```diff
@@ -54,9 +54,7 @@ public ProbesTests(ITestOutputHelper output)
 
     public static IEnumerable<object[]> ProbeTests()
```

Comment:
These tests looks great! Just to make sure, did you also manually verify that the feature works against the Live Debugger app?

Reply:
Yes.

Reply:
Perfect, thank you!